src/main/java/uk/nhs/digital/nhsconnect/nhais/service/FhirToEdifactService.java

```diff
@@ -204,6 +204,8 @@ private OutboundMeshMessage translateInterchange(TranslationItems translationIte
         meshMessage.setContent(edifact);
         meshMessage.setWorkflowId(WorkflowId.REGISTRATION);
         meshMessage.setOperationId(translationItems.operationId);
+        meshMessage.setHaTradingPartnerCode(translationItems.recipient);
+        meshMessage.setMessageSentTimestamp(translationItems.translationTimestamp.toString());
```

Comment:
This will probably be overwritten when publishing to queue

Reply:
yes, a timestamp is lost after publish. It's leftover after testing. Removing this.